### PR TITLE
Board render

### PR DESCRIPTION
**Pull request by jimmacur**
This pull completes iteration 2 with a board rendering method.

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -62,4 +62,21 @@ def place(ship, coordinates)
...
+        # require 'pry'; binding.pry
+        board  = "  1 2 3 4 \n" +
+                 "A #{@cells["A1"].render} #{@cells["A2"].render} #{@cells["A3"].render} #{@cells["A4"].render}\n" +
+                 "B . . . . \n" +
+                 "C . . . . \n" +
+                 "D . . . . \n"
+        
+        p board
+        # render a board that is a string
+        # the first line represents an empty space and the number 1-4
```
Comment (jchirch) [inline lib/board.rb]: let's refactor the commented out code and old pry sessions. Looks gooood!

Comment (jimmacur) [reply lib/board.rb]: Sounds good!

File: lib/board.rb [hunk 2]
```diff
@@ -64,19 +64,20 @@ def place(ship, coordinates)
...
-                 "B . . . . \n" +
-                 "C . . . . \n" +
-                 "D . . . . \n"
-        
+                 "B #{@cells["B1"].render} #{@cells["B2"].render} #{@cells["B3"].render} #{@cells["B4"].render}\n" +
+                 "C #{@cells["C1"].render} #{@cells["C2"].render} #{@cells["C3"].render} #{@cells["C4"].render}\n" +
+                 "D #{@cells["D1"].render} #{@cells["D2"].render} #{@cells["D3"].render} #{@cells["D4"].render}\n"
+        else
+            board  = "  1 2 3 4 \n" +
+                 "A #{@cells["A1"].render(true)} #{@cells["A2"].render(true)} #{@cells["A3"].render(true)} #{@cells["A4"].render(true)}\n" +
```
Comment (jchirch) [inline lib/board.rb:76]: Let's see if there is a way (enumerables?) to make this string interpolated code shorter and more dynamic.

**Review by jchirch — APPROVED** [review]

File: lib/board.rb (hunk 2 above)
Comment (jimmacur) [reply lib/board.rb:76]: I like that idea!

**Pull request merged**